README.md

```diff
@@ -1,28 +1,34 @@
-[![Join the chat at https://gitter.im/zlsa/atc](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zlsa/atc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Travis Production Build State](https://img.shields.io/travis/openscope/openscope/master.svg)](https://github.com/openscope/openscope/tree/master)
+[![openScope Current Release](https://img.shields.io/github/release/openscope/openscope.svg)](https://github.com/openscope/openscope/releases)
+[![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE.md)
+[![Slack Status](http://slack.openscope.co/badge.svg)](http://slack.openscope.co)
+[![Website Status](https://img.shields.io/website-up-down-green-red/http/openscope.co.svg)](http://www.openscope.co)
+
 ## Air Traffic Control
 
 An ATC simulator in HTML5.
 
 ## Running Locally
-It is possible to run this application locally.  You will need to have node and npm install in order for this to work.  For more information, view the [Tools Documentation](tools/README.md)
+
+It is possible to run this application locally. You will need to have [`node`](https://nodejs.org/en/download/) and [`npm`](https://docs.npmjs.com/getting-started/installing-node) installed in order for this to work. For more information, view the [tools documentation](tools/README.md).
 
 ## Privacy
 
-Page hits are tracked with Google Analytics; feel free to use an
-adblocker. The tracking is used only to get the number of page hits.
+Page hits are tracked with Google Analytics, so feel free to use an adblocker. The tracking is used only to get the number of page hits.
 
 ## Contributing
 
-Just create a pull request and I'll get to it. Thanks for helping!
+We use an [GitFlow Branching Model](http://nvie.com/posts/a-successful-git-branching-model) when adding features. If you would like to contribute, please [contact us](https://slack.openscope.co/) and we'll help as soon as we can! Thank you!
+
 ## License
 
 MIT.
 
-Note: the license was formerly CC0; it was set before I was familiar with licensing options.
+**Note:** The license was formerly CC0. It was set before I was familiar with licensing options.
 
-## Exceptions
+### Exceptions
 
-The following files are NOT MIT licensed:
+The following files are **NOT** MIT licensed:
 
 * [`terrain.svg`](http://www.flaticon.com/free-icon/mountain-summit_27798) by [OCHA](http://www.flaticon.com/authors/ocha) is licensed CC BY 3.0.
```

Comment:
negative. that's feature #281 (PR #284). Will go in this sprint, and includes the removal of this section, so it can remain in place in this PR.